lib/WWW/Mechanize.pm

```diff
@@ -1958,21 +1958,59 @@ sub field {
 
 =head2 $mech->select($name, $value)
 
+=head2 $mech->select($name, \%value)
+
+=head2 $mech->select($name, \%values)
```

Comment:
I'm a little unclear on the distinction between `\%value` and `\%values`. From the docs it looks like it's the difference between `{n => 4}` and `{n => [2, 4]}`.

I wonder if there's a way to make this clearer? At first glance I wondered if this was a copy/paste error.